### PR TITLE
API-377: Add GET product model documentation

### PR DESCRIPTION
**Pull request by willy-ahva**

### CODE REVIEW

File: content/akeneo-web-api.yaml [hunk 1]
```diff
@@ -746,6 +746,30 @@ paths:
...
+      description: This endpoint allows you to get the information about a given product model.
+      parameters:
+        - $ref: '#/parameters/code'
+      responses:
+        200:
+          description: OK
+          x-details: Returns the content of the product model in JSON standard format.
+          schema:
+          $ref: '#/definitions/ProductModel'
+      401:
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Indentation is really important. Here, it should be aligned with `200`.

File: content/akeneo-web-api.yaml [hunk 2]
```diff
@@ -3419,6 +3443,123 @@ definitions:
...
+          }
+        ],
+        "name": [
+          {
+            "locale": "en_US",
+            "scope": null,
+            "data": "Biker jacket"
+          }
+        ],
+        "price": [
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Remove this empty price or put a value in it.

File: content/akeneo-web-api.yaml [hunk 3]
```diff
@@ -746,6 +746,30 @@ paths:
...
+        - $ref: '#/parameters/code'
+      responses:
+        200:
+          description: OK
+          x-details: Returns the content of the product model in JSON standard format.
+          schema:
+          $ref: '#/definitions/ProductModel'
+      401:
+        $ref: "#/responses/401Error"
+      406:
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: It's missing the 403 response here.

File: content/akeneo-web-api.yaml [hunk 4]
```diff
@@ -746,6 +746,30 @@ paths:
...
+        - 2.0.1
+      description: This endpoint allows you to get the information about a given product model.
+      parameters:
+        - $ref: '#/parameters/code'
+      responses:
+        200:
+          description: OK
+          x-details: Returns the content of the product model in JSON standard format.
+          schema:
+          $ref: '#/definitions/ProductModel'
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Here also a problem in indentation.

File: content/akeneo-web-api.yaml (hunk 3 above)
Comment (willy-ahva) [reply content/akeneo-web-api.yaml]: 403 is for forbidden access, there is no for the moment

File: content/akeneo-web-api.yaml [hunk 5]
```diff
@@ -796,6 +796,30 @@ paths:
...
+        - $ref: '#/parameters/code'
+      responses:
+        200:
+          description: OK
+          x-details: Returns the content of the product model in JSON standard format.
+          schema:
+            $ref: '#/definitions/ProductModel'
+        401:
+          $ref: "#/responses/401Error"
+        406:
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Still missing the 403 error

File: content/akeneo-web-api.yaml [hunk 6]
```diff
@@ -796,6 +796,30 @@ paths:
...
           $ref: "#/responses/404Error"
 
+  /api/rest/v1/product-models/{code}:
+    get:
+      summary: Get a product model
+      operationId: "get_product_models__code_"
+      tags:
+        - Product models
+      x-versions:
+        - 2.0.1
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Leave it a 2.0 (we won't indicate patches here)

File: content/akeneo-web-api.yaml (hunk 5 above)
Comment (NolwennP) [reply content/akeneo-web-api.yaml]: My bad. You are right. No 403 here since there are neither ACLs nor EE permissions applied for the moment.

**Pull request merged**